pkg/operators/ebpf/extrainfo.go

```diff
@@ -108,8 +123,30 @@ func (i *ebpfInstance) addExtraInfo(gadgetCtx operators.GadgetContext) error {
 		ContentType: "application/json",
 		Content:     []byte(variablesJson),
 	}
+	ebpfInfo.Data["ebpf.relocations"] = &api.GadgetInspectAddendum{
+		ContentType: "application/json",
+		Content:     []byte(relocations),
+	}
 
 	gadgetCtx.SetVar("extraInfo.ebpf", ebpfInfo)
 
 	return nil
 }
+
+func GetRelocations(insns asm.Instructions) (relocations string) {
```

Comment:
* No need to export this function
* Maybe it could be renamed to `getSourceWithRelocations()` since it gets more than just the relocations